clang/utils/TableGen/TableGen.cpp

```diff
@@ -25,6 +25,7 @@ using namespace clang;
 enum ActionType {
   PrintRecords,
   DumpJSON,
+  GenCIRBuiltinsLowering,
```

Comment:
All of the changes in this directory need to be guarded by a `CLANG_ENABLE_CIR` preprocessor check, and we'll need approval from the regular clang maintainers.